calcom/cal.com
# test: add apple calendar testing

- chore: include data-testids
- test: create apple calendar basic test case
- refactor: create fixture to install apple calendar

## What does this PR do?



Fixes #9373
/claim #9373

## Type of change




## How should this be tested?



  - Are there environment variables that should be set?
      - `E2E_TEST_APPLE_CALENDAR_EMAIL`: Your iCloud e-mail
      - `E2E_TEST_APPLE_CALENDAR_PASSWORD`: The app password that you generated on iCloud
        - To generate an app password, go for https://appleid.apple.com/account/manage
        - Open the "App-Specific Password"
        - Click on "Generate an app-specific password" (or in the "+" button if you already have other app passwords)
        - Set a name to this app password
        - Copy the password and paste on the end of your `.env` file setting the `E2E_TEST_APPLE_CALENDAR_PASSWORD` variable
  - What is expected (happy path) to have (input and output)?
    - Paste `yarn e2e apple-calendar.e2e.ts` on your terminal and wait
    - It should successfully login into the Apple Calendar and the test should pass
